admin_manual/contents.rst

```diff
@@ -6,7 +6,7 @@ Table of contents
     :maxdepth: 2
 
     index
-    release_notes
+    release_notes/index
```

Comment:
```suggestion
    upgrade_guide/index
```
Then let's try renaming the folder.
Not sure why it worked locally thou

Reply:
there must be some caching in place. it still points to the old file :see_no_evil:

Reply:
wiping the _build directory and rebuilding makes it work with your changes

Reply:
🎉